src/index.html

```diff
@@ -1,13 +1,101 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Apple A1419"
+        class="card__image"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__product-code">
+        Product code: 195434
+      </div>
+
+
+      <div class="card__raiting">
+        <div class="stars stars--0">
+          <a class="stars__star" href="#stars--1"></a>
+          <a class="stars__star" href="#stars--2"></a>
+          <a class="stars__star" href="#stars--3"></a>
+          <a class="stars__star" href="#stars--4"></a>
+          <a class="stars__star" href="#stars--5"></a>
+        </div>
+
+        <div class="stars stars--card stars--1" id="stars--1">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+
+        <div class="stars stars--card stars--2" id="stars--2">
```

Comment:
Do we need so many stars, can you provide a demo link? Because current doesn't work
![image](https://github.com/mate-academy/layout_product-cards/assets/46566190/ba5f4cc3-6bb6-4ff2-ac17-ed3928945bfe)